hr_commission/models/res_partner.py

```diff
@@ -30,17 +30,17 @@ class ResPartner(models.Model):
         selection_add=[("salesman", "Salesman (employee)")])
     employee = fields.Many2one(
         comodel_name="hr.employee", compute="_get_employee")
-    users = fields.One2many(comodel_name="res.users",
```

Comment:
Why removing this? `user_ids` doesn't contain the same value as this one

Reply:
I don't see the difference, which is it?

https://github.com/OCA/OCB/blob/a23f87326a89849c23531e933488ec760ee27e7c/openerp/addons/base/res/res_partner.py#L305

Reply:
You're right. I got confused. Any way, this should be more focused on recent versions, as we are migrating to them.

Reply:
Well, it's done already, I don't think it hurts to clean this version... :thinking: 

It should be forward-ported, of course.